packages/benchmark-wasm/benchmarks/hover.bench.ts

```diff
@@ -1,11 +1,10 @@
-import { LanguageService } from "../../csslsrs/dist/index.js";
+import { LanguageService } from "csslsrs";
 import { getCSSLanguageService } from "vscode-css-languageservice";
 import { TextDocument } from "vscode-languageserver-textdocument";
 import { bench, describe } from "vitest";
 
-describe("Hover", async () => {
-	const vscodeLanguageService = getCSSLanguageService();
-	const content = `
+const vscodeLanguageService = getCSSLanguageService();
```

Comment:
Just moved some code here so that it's consistent with other benchmarks, found this out while trying to make watch work.